pages/common/csvtool.md

```diff
@@ -0,0 +1,24 @@
+# csvtool
+
+> Utility to filter and extract data from CSV formatted sources.
+> More information: <https://github.com/maroofi/csvtool>.
+
+- Extract the second column from a CSV file:
+
+`csvtool --column 2 {{path/to/file.csv}}`
+
+- Extract the second and fourth columns from a CSV file:
+
+`csvtool --column 2,4 {{path/to/file.csv}}`
+
+- Extract lines from a CSV file where the second column exactly matches 'Foo':
+
+`csvtool --column 2 --search '^Foo$' {{path/to/file.csv}}`
+
+- Extract lines from a CSV file where the second column starts with 'Bar':
+
+`csvtool --column 2 --search '^Bar' {{path/to/file.csv}}`
```

Comment:
How is this example different from the previous one? Perhaps we could mention in the previous one's description that it accepts a regular expression?

Reply:
@sbrl is exact match because `^Foo$` is start and end of the string. here is just starts with `^Bar`

Reply:
Ahhhh, I see. Got it! I always misread the middle of sentences.....

Let's do this then.
```suggestion
`csvtool --column {{2}} --search '{{^Bar}}' {{path/to/file.csv}}`
```